src/client/network.ts

```diff
@@ -165,6 +165,18 @@ export class Request extends ChannelOwner<channels.RequestChannel, channels.Requ
     return this._timing;
   }
 
+  async bodySize(): Promise<number> {
```

Comment:
Should we expose request.sizes instead next to timing to keep it easily accessible?